### ISSUE TITLE
failing to provide host breaks site

### ISSUE THREAD

**Issue by bradfordcondon** [tripal/tripal_elasticsearch#189]
This exception prevents content from loading.

```
Exception: A host was not provided. Please set an Elasticsearch host through the admin interface.
 in ESInstance->__construct() (line 50 of /Users/chet/UTK/tripal/sites/all/modules/custom/tripal_elasticsearch/includes/Elasticsearch/ESInstance.php).
```
Is this as intended?

**Comment by almasaeed2010** [comment]
The construct should not try to load the library if it does not exist

**Comment by almasaeed2010** [comment]
Fixed in #198
